source/include/shadow.h

```diff
@@ -513,6 +597,12 @@ ShadowStatus_t Shadow_GetTopicString( ShadowTopicStringType_t topicType,
  *             null if caller does not need to know the Thing Name contained in the topic string.
  * @param[out] pThingNameLength Pointer to caller-supplied memory for returning the length of the Thing Name,
  *             and can be null if caller does not need to know the Thing Name contained in the topic string.
+ * @param[out] pShadowName Points to the 1st character of Shadow Name inside of the topic string, and can be
+ *             null if caller does not need to know the Shadow Name contained in the topic string. Null is
+ *             returned if the shadow is Classic.
```

Comment:
```suggestion
 *             returned if the shadow is unnamed.
```

I perfer we use unnamed everywhere

Reply:
Sticking with "Classic", per the earlier point.